packages/components/src/DateTimePickers/IconButton/IconButton.scss

```diff
@@ -0,0 +1,15 @@
+@import '../mixins';
+
+.btn {
+    @include nav-action;
+    display: flex;
+    justify-content: center;
+    align-content: center;
+    background: none;
+    border: none;
```

Comment:
Still KO. From my previous review : `Shouldn't that be in nav-action instead of each button style ?`